agents/pfeiffer_tpg366/pfeiffer_tpg366_agent.py

```diff
@@ -34,6 +34,26 @@ def __init__(self, ip_address, port, timeout=10,
         self.comm.connect((self.ip_address, self.port))
         self.comm.settimeout(timeout)
 
+    def channel_power(self):
+        '''
+        Function to check the power status of a given channel
+
+        Args:
+            None
+        Returns:
+            print statement warning about channels being off
+        '''
+        msg = 'SEN\r\n'
```

Comment:
Question about this command. In [the manual](https://www.pfeiffer-vacuum.com/filepool/file/measurement/bg5511ben.pdf?referer=1830&detailPdoId=32066&request_locale=en_US) it looks like state 0 means "gauge cannot be switched on/off" 1 is "gauge is off" and 2 is "gauge is on". Should we be checking for state 1 for seeing if gauges are off? Or am I misunderstanding the command docs?